developers/academy/js/starter_multimodal_data/102_mm_collections/20_create_collection.mdx

```diff
@@ -0,0 +1,89 @@
+
+## <i class="fa-solid fa-code"></i> Code
+
+This example creates a collection for the movie data:
+
+<FilteredTextBlock
+  text={TSCode}
+  startMarker="// CreateMovieCollection"
+  endMarker="// END CreateMovieCollection"
+  language="ts"
+/>
+
+Each collection definition must have a name. Then, you can define additional parameters like we've done in this example.
+
+## <i class="fa-solid fa-chalkboard"></i> Explain the code
+
+### <i class="fa-solid fa-chalkboard"></i> Properties
+
+Properties are the object attributes that you want to store in the collection. Each property has a name and a data type.
+
+In our movie database, we have properties like `title`, `release_date` and `genre_ids`, with data types like `TEXT` (string), `DATE` (date), or `INT` (integer). It's also possible to have arrays of integers, like we have with `genre_ids`.
+
+As a multimodal object, we also have the `poster` property which is the image data, which is saved as a `BLOB` (binary large object) data type.
+
+#### Auto-schema
+
+Weaviate can automatically [infer the schema](/developers/weaviate/config-refs/schema/index.md#auto-schema) from the data. However, it's a good practice to define the properties explicitly, for better control and to avoid surprises.
+
+### <i class="fa-solid fa-chalkboard"></i> Vectorizer configuration
+
+If you do not specify the vector yourself, Weaviate will use a specified vectorizer to generate vector embeddings from your data.
+
+In this code example, we specify the `multi2vec-cohere` module. This module uses the Cohere Multimodal Embed 3 model to generate vector embeddings from the text and image data.
+
+You can specify any number of text and image properties to be used for vectorization, and weight them differently. The weights are used to determine the relative importance of each property in the vector embedding generation process. In this example, we vectorize the `poster` property (an image) with a 90% weight and the `title` property (a string) with a 10% weight.
+
+<FilteredTextBlock
+  text={TSCode}
+  startMarker="// Define the vectorizer module"
+  endMarker="// Define the generative module"
+  language="ts"
+/>
+
+### <i class="fa-solid fa-chalkboard"></i> Generative configuration
+
+If you wish to use your collection with a generative model (e.g. a large language model), you must specify the generative module.
+
+In this code example, we specify the `cohere` module (`generative-cohere` is the full name) with default options.
+
+<FilteredTextBlock
+  text={TSCode}
+  startMarker="// Define the generative module"
+  endMarker="// END generativeDefinition"
+  language="ts"
+/>
+
+import MutableGenerativeConfig from '/_includes/mutable-generative-config.md';
+
+<MutableGenerativeConfig />
+
+### <i class="fa-solid fa-code"></i> Javascript classes
+
+The code example makes use of methods such as `property`, `dataType` and `configure`. They are defined in the `weaviate.configure` object and are used to define the collection.
```

Comment:
I think this section needs a review.

The heading is called "classes", but we talk about "methods". Also I don't see any classes/methods called "property" in the code, although I to see a "dataType" and "configure". 

I don't know what the terminology should be exactly, but maybe it could be "object" or something. I did a quick search and they don't seem like classes and methods.